src/main/java/stirling/software/SPDF/model/ApplicationProperties.java

```diff
@@ -309,4 +311,92 @@ public String getProducer() {
             }
         }
     }
+
+    @Data
+    public static class SemaphoreLimit {
```

Comment:
I think these are so closely related they can fall under the same parent setting
ProcessExecutor might be best naming
ProcessExecutor.TimeoutMinutes etc
also for properties not many users will know what SemaphoreLimit  means
perhaps rename to session-limit or something more layman

Reply:
Will do! Thanks for the feedback!